packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -227,6 +245,13 @@ class ScrollDragController implements Drag {
   /// Called when [dispose] is called.
   final VoidCallback onDragCanceled;
 
+  /// Velocity that was present from a previous [ScrollActivity] when this
+  /// drag began.
```

Comment:
nit: could this wrap up to the previous line?

Reply:
I wrapped the other stuff in this file to fit 80 chars per line instead :)